packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isPickButtonEnabled.st

```diff
@@ -0,0 +1,4 @@
+buttons
+isPickButtonEnabled
```

Comment:
once again I'm not sure about the name.... is this needed for the button or why do we have this as an extra method?

Reply:
yep, the enabling of the button is determined by the function's result